store/tikv/lock_resolver.go

```diff
@@ -370,7 +373,7 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			return status, err
 		}
 		if cmdResp.CommitVersion != 0 {
-			status = TxnStatus(cmdResp.GetCommitVersion())
+			status = TxnStatus{0, cmdResp.GetCommitVersion()}
```

Comment:
`ttl` is always `0`? change this later or?

Reply:
In the `CleanUp` proto, the result would be either committed or rollbacked, so the `ttl` is always 0.